app/models/user.rb

```diff
@@ -36,12 +39,33 @@ def remember
     update_attribute :remember_digest, User.digest(remember_token)
   end
 
-  def authenticated? remember_token
-    return false unless remember_digest
-    BCrypt::Password.new(remember_digest).is_password?remember_token
-  end
+  def authenticated? attribute, token
+    digest = send "#{attribute}_digest"
```

Comment:
Can I ask you a question, what is send method ?

Reply:
Send method allow us to call a method